package/sbp_rtcm3_bridge/src/sbp_rtcm3_bridge.c

```diff
@@ -25,25 +25,37 @@
 #define PROGRAM_NAME "sbp_rtcm3_bridge"
 
 #define RTCM3_SUB_ENDPOINT "ipc:///var/run/sockets/rtcm3_internal.pub" /* RTCM3 Internal Out */
+#define RTCM3_PUB_ENDPOINT "ipc:///var/run/sockets/rtcm3_internal.sub" /* RTCM3 Internal In */
 
 bool rtcm3_debug = false;
 
-struct rtcm3_sbp_state state;
+struct rtcm3_sbp_state rtcm3_sbp_state;
```

Comment:
`rtcm3_sbp_state` != `sbp_rtcm3_state` ?! prob need to clarify naming b/w these two, maybe just need to insert a `_to_` there

Reply:
done - tried to clarify the naming on each